frontend/public/components/start-guide.tsx

```diff
@@ -98,13 +86,11 @@ export const withStartGuide = (WrappedComponent, doNotDisable: boolean = false)
 /* eslint-disable no-unused-vars, no-undef */
 type StartGuideProps = {
   dismissible?: boolean;
+  localStorageKey?: string;
+  startGuide?: any;
```

Comment:
`startGuide` needs to be required.

```suggestion
  startGuide: React.ReactNode;
```